lib/bloc/account/account_bloc.dart

```diff
@@ -138,10 +138,7 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
     _log.info("lnurlWithdraw amount: $amountSats, description: '$description', reqData: $reqData");
     try {
       return await _breezLib.lnurlWithdraw(
-        amountSats: amountSats,
-        reqData: reqData,
-        description: description,
-      );
+          request: sdk.LnUrlWithdrawRequest(amountMsat: amountSats, data: reqData, description: description));
```

Comment:
`reqData` can be renamed to `data` for this function and please add a comma after `description` and `req` object then reformat for readability.

Reply:
Done